docs/reference/glossary.md

```diff
@@ -1,12 +1,23 @@
 # Glossary
 
-Since so many terms are thrown around in the documentation and comments within the codebase below is a short glossary to ensure everyone is talking about the same thing.
-
-* Original Atom Server / Atom.io Server: The Backend Package Repository Server that was built, maintianed, and sunset by Atom/GitHub/Microsoft. This server we have no access to, and has no source available. We are only able to learn about it by interactions with it, and the interactions baked into the source code of tools that rely on it.
-* Atom Backend / Pulsar Backend: The new Open Source creation of the Package Repository Server. Where within Pulsar environments it's referred to simply as the Pulsar Package Repository.
-* Server Status Object: The object most modules return within in the codebase, more details are [available](./bubbled_errors.md).
-* Package Object Full: The largest form of Package data returned to end users, more details are [available](./returns.md#packageobjectfull).
-* Package Object Short: The most common form of Package data returned to end users, more details are [available](./returns.md#packageobjectshort).
-* Package Object Mini: The smallest possible return of Package data returned to end users, more details are [available](./returns.md#packageobjectmini).
-* User Object: The only form of data returned to end users that contains user data, more details are [available](./returns.md#userobject).
-* Bubbled Errors: The act of letting an error be easily caught and letting it bubble up from lower level modules back to the end users to ensure they see a proper report of what went wrong, more details are [available](./bubbled_errors.md).
+This document provides definitions of words and terms used within this projects source code and documentation that cannot be easily found elsewhere.
+
+* **Atom Package Registry (APR)**: The backend Package Reigstry server that was built, maintained, and sunet by Atom/GitHub/Microsoft. Pulsar cannot access this server, or it's code base in any way. Pulsar has only been able to learn about it through interactions with it, and the interactions that are baked into the source code of tools that rely on the APR. In some places, within the source code/documentation, you may find alternative names used to refer to the APR eg. Original Atom Server, Atom.io Server, Atom's Backend.
+
+* **Bubbled Errors**: Refers to the behavior used, that allows errors to bubble up from lower level modules to the HTTP Handling layer, to ensure the end user always sees exactly how and why something has failed. Additional details on [Bubbled Errors]().
+
+* **Package Object Full**: Refers to the largest format of a package's data that is returned to end users within the PPR. Containing the full range of information stored about a single package, such as it's base information, as well as details of every single version of that package. Additional details on a [Package Object Full]().
```

Comment:
`PPR` isn't defined within in this doc. The first instance should be `Pulsar Package Registry (PPR)`.

Reply:
Great catch, and really the glossary probably shouldn't use any internal acronyms, as it's unlikely a reader will fully read through the glossary themselves